src/util/logger/Logger.cpp

```diff
@@ -39,7 +41,16 @@ static string get_worker_name() {
 
 void Logger::log(std::string message, const std::string log_type) {
     pthread_t tid = pthread_self();
-    message = "[" + worker_name + " : " + to_string(tid) + "] " + message;
+
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
+    long millis = tv.tv_sec * 1000 + tv.tv_usec / 1000;
+    // TODO: temporary fix only.
+    cout << " [" << millis << "] [" << log_type << "] [" << worker_name << " : " << getpid() << ":" << tid << "] "
```

Comment:
Remove the `cout`

Reply:
It is needed, otherwise the logger will not log.

Reply:
I see. So we do not use spdlog functionality?

Reply:
It hangs after fork (only on workers)

Reply:
I see